cmd/util/ledger/migrations/storage-fees-migration.go

```diff
@@ -0,0 +1,79 @@
+package migrations
+
+import (
+	"github.com/onflow/flow-go/engine/execution/state"
+	fvm "github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/utils"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// iterates through registers keeping a map of register sizes
+// after it has reached the end it add storage used and storage capacity for each address
+func StorageFeesMigration(payload []ledger.Payload) ([]ledger.Payload, error) {
+	storageUsed := make(map[string]uint64)
+	newPayload := make([]ledger.Payload, len(payload))
+
+	for i, p := range payload {
+		err := process(p, storageUsed)
+		if err != nil {
+			return nil, err
+		}
+		newPayload[i] = p
+	}
+
+	for s, u := range storageUsed {
+		storageUsedByStorageUsed := fvm.RegisterSize(
+			flow.BytesToAddress([]byte(s)),
+			false, "storage_used",
+			make([]byte, 8))
+		u = u + uint64(storageUsedByStorageUsed)
+
+		newPayload = append(newPayload, ledger.Payload{
+			Key:   makeKey(s, "storage_used"),
+			Value: utils.Uint64ToBinary(u),
+		})
+	}
+	return newPayload, nil
+}
+
+func makeKey(owner string, key string) ledger.Key {
```

Comment:
This seems like a common function for all migrations. Can it live in a different file?